Indicators/AccelerationBands.cs

```diff
@@ -0,0 +1,116 @@
+﻿using QuantConnect.Data.Market;
+
+namespace QuantConnect.Indicators
+{
+    /// <summary>
+    ///     The Acceleration Bands created by Price Headley plots upper and lower envelope bands around a moving average.
+    /// </summary>
+    /// <seealso cref="QuantConnect.Indicators.IndicatorBase{QuantConnect.Data.Market.TradeBar}" />
+    public class AccelerationBands : IndicatorBase<TradeBar>
+    {
+        private readonly decimal _width;
+
+        /// <summary>
+        ///     Gets the type of moving average
+        /// </summary>
+        public MovingAverageType MovingAverageType { get; }
+
+        /// <summary>
+        ///     Gets the middle acceleration band (moving average)
+        /// </summary>
+        public IndicatorBase<IndicatorDataPoint> MiddleBand { get; }
+
+        /// <summary>
+        ///     Gets the upper acceleration band  (High * ( 1 + Width * (High - Low) / (High + Low)))
+        /// </summary>
+        public IndicatorBase<IndicatorDataPoint> UpperBand { get; }
+
+        /// <summary>
+        ///     Gets the lower acceleration band  (Low * (1 - Width * (High - Low)/ (High + Low)))
+        /// </summary>
+        public IndicatorBase<IndicatorDataPoint> LowerBand { get; }
```

Comment:
all of these could be readonly?

Reply:
Yes, but I followed the same criteria used in the BolligerBands indicator.